### DISCUSSION TITLE
Welcome to Contact-List-App Discussions!

### DISCUSSION THREAD

**Discussion by vishal-rathod-07** [vishal-rathod-07/Contact-List-App#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SujalXplores** [comment]
This app can be optimized by

- add SEO to the page
- add styled components
- add typescript support for typesafe

**Reply by SujalXplores** [reply]
I can raise 1 PR that will include all the **changes**

**Reply by SujalXplores** [reply]
- also all `package.json` dependencies should be to latest version
migration needed
